tests/test_schema.py

```diff
@@ -469,6 +473,47 @@ def test_schema_not_supported_type_issue_goodatbles_304():
     assert schema.fields[1] == {"name": "age", "type": "bad"}
 
 
+unzipped_dir = "tests/fixtures/output-unzipped"
+
+
+@pytest.mark.parametrize(
+    "zip_path",
+    [
+        str((Path(base) / file).relative_to(unzipped_dir))
```

Comment:
@shashigharti 
I meant rewriting this like to somethin like:
Path('/'.join(Path(base), file)).relative_to

As ZIP uses "/" even on windows

Reply:
sure I will change that!. I forgot to update that part.